src/test/java/com/jcraft/jsch/JSchTest.java

```diff
@@ -64,6 +67,26 @@ void checkLoggerBehavior() throws Exception {
     assertSame(JSch.DEVNULL, jsch.getInstanceLogger(), "instance logger should be DEVNULL");
   }
 
+  @Test
+  void getConfig() throws Exception {
```

Comment:
SonarQube points out that no method in this test case throws a checked Exception, so can we drop the `throws Exception` clause?

Reply:
all the other tests does the same

Reply:
Yes, I'll put on my TODO list to clean that up.